write-fonts/src/font_builder.rs

```diff
@@ -21,6 +21,11 @@ impl<'a> FontBuilder<'a> {
         self
     }
 
+    /// Returns `true` if the builder contains a table with this tag.
```

Comment:
Comment adds nothing to type signature, suggest removal.

Reply:
I think in general under-documentation is a far greater problem than over-documentation, and requiring the documentation of all public items is something that we can (and I think we should) do in CI. 🤷